pkg/clients/rorclient/transports/resttransport/v2/resources/resources.go

```diff
@@ -44,24 +45,35 @@ func (c *V2Client) Update(res *rorresources.ResourceSet) (*rorresources.Resource
 	return ret, nil
 }
 
-func (c *V2Client) GetByUid(uid string) (rorresources.ResourceSet, error) {
-	// implementation goes here
-	return rorresources.ResourceSet{}, nil
-}
+func (c *V2Client) Delete(uid string) (*rorresources.ResourceUpdateResults, error) {
+	var out rorresources.ResourceUpdateResults
 
-func (c *V2Client) UpdateByUid(uid string, res *rorresources.ResourceSet) (string, error) {
-	// implementation goes here
-	return "", nil
-}
+	url, err := url.JoinPath(c.basePath, "uid", uid)
+	if err != nil {
+		return nil, fmt.Errorf("could not create url: %w", err)
+	}
+
+	err = c.Client.Delete(url, &out)
+	if err != nil {
+		return nil, err
+	}
 
-func (c *V2Client) DeleteByUid(uid string) (string, error) {
-	// implementation goes here
-	return "", nil
+	return &out, nil
 }
 
-func (c *V2Client) ExistsByUid(uid string) (bool, error) {
-	// implementation goes here
-	return false, nil
+func (c *V2Client) Exists(uid string) (bool, error) {
+	//url, err := url.JoinPath(c.basePath, "uid", uid)
```

Comment:
Still not implemented?

Reply:
Il get to it today, i just wanted to get the PR up